src/Geta.Bring/Shipping/Model/AgreementPrice.cs

```
@@ -0,0 +1,44 @@
+﻿namespace Geta.Bring.Shipping.Model
+{
+    public class AgreementPrice
+    {
+        public AgreementPrice(
+            string agreementName, 
+            int agreementNumber,
+            string subAgreementName,
+            int subAgreementNumber,
+            double price)
+        {
+            AgreementName = agreementName;
```

Add null check for mandatory arguments so it fails fast.